pkgs/development/python-modules/robotframework-python3/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchPypi, buildPythonPackage, six }:
+
+with stdenv.lib;
+
+buildPythonPackage rec {
+  name  = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.